modules/navier_stokes/src/fvkernels/INSFVMomentumAdvection.C

```diff
@@ -445,6 +445,7 @@ INSFVMomentumAdvection::interpolate(Moose::FV::InterpMethod m,
                   ? elem_one == &_face_info->elem() && elem_two == _face_info->neighborPtr()
                   : elem_one == _face_info->neighborPtr() && elem_two == &_face_info->elem(),
               "The determineElemOneAndTwo utility determined the wrong value for elem_is_elem_one");
+  mooseAssert(elem_is_elem_one, "FVFluxKernel elem should coincide with the faceInfo elem.");
```

Comment:
Since this appears to be true, let's move it ahead of the assert on line 444, and then remove the else branch of the ternary

Reply:
We could just delete all the ternaries and set elem_one to face_info->elem
I guess there's no rush to do that. Only if needed for optimization

Reply:
No you are right. Now that it's clearly demonstrated that `elem_one` is always `_face_info->elem()`, we should just remove the call to `Moose::FV::determineElemOneAndTwo` and then remove the associated asserts.

Reply:
Ok I ll do that and add a block restricted INSFV test too